### PR TITLE
feat: rainbowkit v2

### PR DESCRIPTION
**Pull request by sanyu1225**
## Related Links

**Asana**: https://app.asana.com/0/1201812548509877/1206295937878532/f

## Checklist

- [X] Pasted Asana link.
- [X] Tagged labels.

### CODE REVIEW

File: .changeset/smart-emus-pretend.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@blocto/rainbowkit-connector': patch
```
Comment (mordochi) [inline .changeset/smart-emus-pretend.md:2]: 是不是也跟著大家的版本升 major ？

Comment (sanyu1225) [reply .changeset/smart-emus-pretend.md:2]: https://github.com/blocto/blocto-sdk/pull/411 我先把v1 major發布 這邊在改成major 😢

Comment (sanyu1225) [reply .changeset/smart-emus-pretend.md]: [b05551d](https://github.com/blocto/blocto-sdk/pull/410/commits/b05551d9e9d5cc86d7d900b222e7838c3c475efa) 改嚕

File: adapters/rainbowkit-connector/src/index.ts [hunk 2]
```diff
@@ -1,29 +1,28 @@
...
-    ios: 'https://apps.apple.com/app/blocto/id1481181682',
-    android: 'https://play.google.com/store/apps/details?id=com.portto.blocto',
-  },
-  installed: true,
-  createConnector: (): any => {
-    const connector = new BloctoConnector({ chains, options: { appId } });
+export const bloctoWallet =
+  ({ appId } = { appId: undefined }) =>
+  (): Wallet => {
+    console.log('appId: ', appId);
```
Comment (mordochi) [inline adapters/rainbowkit-connector/src/index.ts:7]: console.log :i_see_what_you_did_there:

**Review by mordochi — APPROVED** [review]

File: adapters/rainbowkit-connector/src/index.ts (hunk 2 above)
Comment (sanyu1225) [reply adapters/rainbowkit-connector/src/index.ts]: 🔧 fixed [027cef6](https://github.com/blocto/blocto-sdk/pull/410/commits/027cef6cad6c10ff549a4146b88936757fcdf926)

**Review by mordochi — APPROVED** [review]

**Pull request merged**